modules/installation-dns-bare-metal.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+
+[id="installation-dns-bare-metal-{context}"]
+= Bare metal DNS requirements
+
+The following DNS records are required for an {product-title} cluster. In each
+record, `<cluster_name>` is the cluster name and `<base_domain>` is the cluster
+base domain that you specify in the `install-config.yaml` file.
+
+.Required DNS records
+[cols="1a,3a,5a",options="header"]
+|===
+
+|Component
+|Record
+|Description
+
+|Kubernetes API
+|`api.<cluster_name>.<base_domain>`
```

Comment:
changes in flight https://github.com/openshift/installer/pull/1605/files#diff-1a64e7030769df4d25b1c09ddb8f62de

Reply:
I'm updating the PR based on the current change and following it in case it changes again.